modules/FIC_FileHandler.py

```diff
@@ -30,11 +33,21 @@ def save(self, directory, file_name, content):
         if file_name.endswith(".md"):
             pass
         elif file_name.endswith(".json"):
-            import json
             with open(path, "w") as json_file:
                 json.dump(content, json_file, indent=2)
 
         else:
             print("File Extension is not supported.")
             exit(2)
         pass
+
+    def load_repos(self):
+        from modules.config import REPOSITORIES_FILE
+        import json
+        self.repos_container = json.loads(open(REPOSITORIES_FILE).read())
```

Comment:
Since you started to work on this, we now have `self._construct_path()` please implement it in the open() function